Day3/checkoutform.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
+</head>
+<style>
+    img {
+        display: inline;
+        height: 20px;
+    }
+
+    /* .container{
+         border: 2px solid;
+         width: 500px;
+    }
+    .bill{
+        border: 1px solid;
+        width:250px ;
+        height: 300px;
+    } */
+    .bill,
+    .Payment {
+        display: inline-block;
+
+    }
+</style>
+
+<body>
+    <h2>Responsive Checkout Form</h2>
+    <div class="container">
+        <form action="">
+            <div class="bill">
+                <h2>Billing Address</h2>
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                    class="bi bi-person-fill" viewBox="0 0 16 16">
+                    <path d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3Zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6Z" />
```

Comment:
no need of svg elements, instead use icon classes like `btn-`
check this link for https://icons.getbootstrap.com/#install adding CDN link